Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -4394,12 +4394,10 @@ bool CClientGame::ApplyPedDamageFromGame(eWeaponType weaponUsed, float fDamage,
             return false;
         }
 
-        if (pDamagedPed->IsLocalPlayer())
```

Comment:
Wasn't this check added to avoid possible desync issues?

Reply:
I don't think so, since HP will be synchronized when using setElementHealth in onClientPedDamage

Reply:
HP is synchronized, but a remote player can die locally. It can cause nasty issues potentially